wasmbinding/stargate_whitelist.go

```diff
@@ -17,8 +29,83 @@ import (
 var StargateWhitelist sync.Map
 
 func init() {
+	// cosmos-sdk queries
+
+	// auth
 	StargateWhitelist.Store("/cosmos.auth.v1beta1.Query/Account", &authtypes.QueryAccountResponse{})
```

Comment:
I wish there was a way to infer these w/o hard-coding them :-/

Reply:
https://github.com/osmosis-labs/osmosis/issues/2627

Reply:
This should be possible with the chnages being done in the sdk to support deterministic queries.

Reply:
hi @marbar3778, can you give me the link about the changes you said, thanks!

Reply:
https://github.com/cosmos/cosmos-sdk/issues/13041